client/my-sites/plan-features/index.jsx

```diff
@@ -459,20 +458,24 @@ PlanFeatures.defaultProps = {
 export default connect(
 	( state, ownProps ) => {
 		const { isInSignup, placeholder, plans, onUpgradeClick } = ownProps;
+		const selectedSiteId = isInSignup ? null : getSelectedSiteId( state );
+		const sitePlan = selectedSiteId && getSitePlan( state, selectedSiteId );
+		const hasSitePlanOwner = sitePlan && sitePlan.owner_id > 0;
```

Comment:
I think this would be useful enough to track on the current user state tree like we do for [user currency](https://github.com/Automattic/wp-calypso/blob/master/client/state/current-user/reducer.js#L44). 

For example the selector might look like:
```javascript
isCurrentUserSiteOwner( state, siteId );
```